### PR TITLE
New Exposure and Lighting Defaults (and calibrate examples)

### LINKED ISSUE #11577: default point lights seems wrong after exposure
## Bevy version

Main after https://github.com/bevyengine/bevy/pull/11347

## What went wrong

Looking at examples, most point lights are set to 150000 lumens, which is more than direct sunlight. Some are even up to 1000000 lumens, which seems an absurd value to set.

The default value of a point light is 800 lumens, which is now useless.

Bevy default values should make sense, and examples should not use values that don't make sense.

### PR DESCRIPTION
**Pull request by cart**
# Objective

After adding configurable exposure, we set the default ev100 value to `7` (indoor). This brought us out of sync with Blender's configuration and defaults. This PR changes the default to `9.7` (bright indoor or very overcast outdoors), as I calibrated in #11577. This feels like a very reasonable default.

The other changes generally center around tweaking Bevy's lighting defaults and examples to play nicely with this number, alongside a few other tweaks and improvements. 

Note that for artistic reasons I have reverted some examples, which changed to directional lights in #11581, back to point lights.
 
Fixes #11577 

---

## Changelog

- Changed `Exposure::ev100` from `7` to `9.7` to better match Blender
- Renamed `ExposureSettings` to `Exposure`
- `Camera3dBundle` now includes `Exposure` for discoverability
- Bumped `FULL_DAYLIGHT ` and `DIRECT_SUNLIGHT` to represent the middle-to-top of those ranges instead of near the bottom
- Added new `AMBIENT_DAYLIGHT` constant and set that as the new `DirectionalLight` default illuminance. 
- `PointLight` and `SpotLight` now have a default `intensity` of 1,000,000 lumens. This makes them actually useful in the context of the new "semi-outdoor" exposure and puts them in the "cinema lighting" category instead of the "common household light" category. They are also reasonably close to the Blender default.
- `AmbientLight` default has been bumped from `20` to `80`.

## Migration Guide

- The increased `Exposure::ev100` means that all existing 3D lighting will need to be adjusted to match (DirectionalLights, PointLights,  SpotLights, EnvironmentMapLights, etc). Or alternatively, you can adjust the `Exposure::ev100` on your cameras to work nicely with your current lighting values. If you are currently relying on default intensity values, you might need to change the intensity to achieve the same effect. Note that in Bevy 0.12, point/spot lights had a different hard coded ev100 value than directional lights. In Bevy 0.13, they use the same ev100, so if you have both in your scene, the _scale_ between these light types has changed and you will likely need to adjust one or both of them.

### CODE REVIEW

**Comment by superdump** [comment]
Maybe worth noting as @fintelia wrote on discord that a the exposure was configurable, point/spot lights had one ev100 value and directional had another, so when using a combination of point/spot and directional lights in a scene you can’t just adjust the ev100 to achieve the old behaviour, you would have to adjust either all lights, or the ev100 and some lights.

**Review by superdump — APPROVED** [review]
Code-wise it all looks fine. The reasoning behind the changes and new defaults is fine.

I do think we should clearly note how things worked before and how they work now to help people migrate their configurations and to think in the new way. Maybe that was done in the PR that introduced exposure.

File: examples/window/screenshot.rs [hunk 1]
```diff
@@ -47,12 +47,8 @@ fn setup(
         ..default()
     });
     // light
-    commands.spawn(DirectionalLightBundle {
-        directional_light: DirectionalLight {
-            shadows_enabled: true,
```
Comment (GitGhillie) [inline examples/window/screenshot.rs:52]: `screenshot` example lost its shadows

**Review by GitGhillie — COMMENTED** [review]
IMO the ambient light is generally too dark with this PR. Easiest to see in animated_fox but also in simple examples like 3d_scene.
0.12 vs this PR:
image

**Comment by GitGhillie** [comment]
Alien_cake_addict is also looking a bit grim

File: crates/bevy_render/src/camera/camera.rs [hunk 2]
```diff
@@ -90,12 +90,12 @@ pub struct ComputedCameraValues {
 ///
 /// <https://en.wikipedia.org/wiki/Exposure_(photography)>
 #[derive(Component)]
-pub struct ExposureSettings {
+pub struct Exposure {
```
Comment (rparrett) [inline crates/bevy_render/src/camera/camera.rs:93]: This could use a mention in the migration guide

**Comment by rparrett** [comment]
Some notes on examples not mentioned yet:

```
custom_skinned_mesh - more ambient light?
gltf_skinned_mesh - more ambient light?
spotlight - more ambient light?
post_processing - too bright
resizing - super dim
minimising - super dim
```

File: crates/bevy_render/src/camera/camera.rs (hunk 2 above)
Comment (DGriffin91) [reply crates/bevy_render/src/camera/camera.rs:93]: ExposureSettings didn't exist in 0.12

Comment (rparrett) [reply crates/bevy_render/src/camera/camera.rs:93]: Right, oops.

Comment (GitGhillie) [reply crates/bevy_render/src/camera/camera.rs:93]: The migration guide of 11347 would need to be adjusted though, not sure how that's usually handled

Comment (DGriffin91) [reply crates/bevy_render/src/camera/camera.rs:93]: Good point

**Comment by cart** [comment]
Tweaked everything as suggested and bumped up the default AmbientLight intensity slightly (enough for it to more clearly register while still being reasonably subtle)

**Review by GitGhillie — APPROVED** [review]
LGTM

**Pull request merged**